components/HowTo.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  Flex,
+  Heading,
+  OrderedList,
+  ListItem,
+  Text,
+  Link,
+} from "@chakra-ui/layout";
+
+// TODO: Copy is a placeholder and pulled from their docs
+const HowTo = () => {
+  return (
+    <Flex
+      direction="column"
+      border="1px solid"
+      borderRadius="0.3rem"
+      padding="2rem"
+      maxWidth="40rem"
+      marginTop="3rem"
+      marginBottom="3rem"
+    >
+      <Heading as="h1" size="lg" alignSelf="center">
+        Tutorial
+      </Heading>
+      <Heading as="h4" size="sm" marginTop="1rem">
+        How to shield funds?
+      </Heading>
+      <OrderedList marginTop="1rem">
+        <ListItem>
+          Create a Railgun wallet on{" "}
+          <Link href="https://app.railway.xyz/" isExternal>
+            Railway
+          </Link>
```

Comment:
One nit is the links currently have no style until you hover, so you can't tell it's a link. Up to you if you want to fix in this PR or track in an issue

<img width="287" alt="image" src="https://user-images.githubusercontent.com/17163988/211637944-3dfe8a58-dd36-42b0-806a-528fdd82fdee.png">

Reply:
Yes, good call, I will create a ticket and we can handle styling towards the end. Does that sound good @wildmolasses

Reply:
#33